Library/Homebrew/cask/quarantine.rb

```diff
@@ -55,26 +55,58 @@ def status(file)
                      print_stderr: false).stdout.rstrip
     end
 
-    def cask(cask: nil, download_path: nil)
+    def disable_translocation!(xattr)
```

Comment:
Maybe name this `delete_translocation_bit`.

Reply:
`delete_translocation_bit` would be misleading because it’s actually set, not cleared. `set_translocation_bit` may also be misleading because it’s really a do-not-translocate bit.

@reitermarkus I’d be cool with `set_no_translocation_bit!`, in case you’d prefer that over `disable_translocation!`.

Reply:
Ok, `set_no_translocation_bit` then. (Without the `!`, because it doesn't really do anything destructive in the idiomatic Ruby sense.)

Reply:
Rubocop complains:
```
Offenses:

Library/Homebrew/cask/quarantine.rb:58:9: C: Do not prefix writer method names with set_.
    def set_no_translocation_bit(xattr)
        ^^^^^^^^^^^^^^^^^^^^^^^^

690 files inspected, 1 offense detected
```

I've changed it to `toggle_no_translocation_bit`, please let me know if you're fine with this.

Reply:
`toggle` is also misleading. It's fine to explicitly disable RuboCop for this line because it's not a writer method.